core/model-api/src/main/java/org/eclipse/rdf4j/model/ValueFactory.java

```diff
@@ -30,7 +27,7 @@
 	 *
 	 * @param iri A string-representation of a IRI.
 	 * @return An object representing the IRI.
-	 * @throws IlllegalArgumentException If the supplied string does not resolve to a legal (absolute) IRI.
+	 * @throws IllegalArgumentException If the supplied string does not resolve to a legal (absolute) IRI.
```

Comment:
good catch. I had to look several times before I even realized what you had changed.